frog/imports/ui/Body.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 
 import Repository from './repository/Repository.jsx';
-import Editor from './Editor.jsx';
+import Editor from './Editor/Editor.jsx';
```

Comment:
Rename the folder `imports/ui/Editor` to `imports/ui/editor` to be coherent with `imports/ui/repository`